script/release/publish-to-npm.js

```diff
@@ -161,6 +162,26 @@ new Promise((resolve, reject) => {
       }
     }
   })
+  .then(() => {
+    const DEPSContent = fs.readFileSync(path.resolve(__dirname, '..', '..', 'DEPS'), 'utf8');
```

Comment:
Perhaps it would be better to use the code written by @RaisinTen  in https://github.com/electron/electron/issues/28006#issuecomment-1035985137 here. That way we don't have to parse the `DEPS` file.

Moreover, electron's CI probably solves the related system dependency problem.